docs/source/NEWS.rst

```diff
@@ -6,6 +6,7 @@ Changelog
 
 - fix ``ModuleNotFoundError: No module named 'cStringIO'`` in ldaptor-ldap2pdns.
 - move scripts to console_scripts entry_points
+- replace deprecated calls to ``base64.decodestring`` and ``base64.encodestring``.
```

Comment:
I think it's worth including the changelog - as it makes the code work on Py3.9. But I don't want to advertise py3.9 support without CI support: https://github.com/twisted/ldaptor/pull/210

```suggestion
- replace deprecated calls to ``base64.decodestring`` and ``base64.encodestring``,
  to support Python 3.9. Official support for Python 3.9 pending a new twisted release.
```

Reply:
I would say that this PR could have been merged without a release note.

In the #210 PR the release note can read: 'Ldaptor now supports Python 3.9'

But is ok. Thanks